### PR TITLE
[Tests] Refactor and remove .kui* classes in tests

### LINKED ISSUE #3387: Refactor usage of `.kui*` classes in tests

### PR DESCRIPTION
**Pull request by Aigerim-ai**
### Description

Refactoring .kui*  classes in tests
### Issues Resolved

 helps to fix #3387 first part which includes adding test-id and changing test case
### Check List
- [x] All tests pass
  - [ ] `yarn test:jest`
  - [ ] `yarn test:jest_integration`
  - [ ] `yarn test:ftr`
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [ ] Update [CHANGELOG.md](./../CHANGELOG.md)
- [ ] Commits are signed per the DCO using --signoff

### CODE REVIEW

**Review by joshuarrrr — COMMENTED** [review]
@Aigerim-ai As we discussed, the change in `timeline_help` is not necessary. As for the `listing_table` changes, we should trace back usage to see if:

1. `getAllItemsNamesOnCurrentPage` is actually used anywhere
2. `ListingTableProvider` is actually being used or called by any tests

Depending on the findings we may be able to either remove the method or the provider altogether, because they appear to be referencing components that are no longer in use.

**Comment by joshuarrrr** [comment]
You'll also need to add a CHANGELOG entry under the Test section.

**Comment by Aigerim-ai** [comment]
Hello! @joshuarrrr 
Regarding the the changes my next updated on this PR will include
1. I have returned previous state of timeline_help
2. getAllItemsNamesOnCurrentPage is not used anywhere besides getAllItemsNames which is not used anywhere, both functions are in listing_table file -> as you suggested I am removing both of them.
3. ListingTableProvider is called in test/functional/services/index.ts, for convention we declare all services in one file listingTable:ListingTableProvider and then we call const listingTable = getService('listingTable') in files by using getService prop;
4. Added CHANGELOG.md

File: CHANGELOG.md [hunk 1]
```diff
@@ -165,7 +165,7 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 - [Vis Builder] Adds field unit tests ([#3211](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3211))
 - [BWC Tests] Add BWC tests for 2.6.0 ([#3356](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3356))
 - Prevent primitive linting limitations from being applied to unit tests found under `src/setup_node_env` ([#3403](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3403))
-
+- Removed functions which are no longer in use([#3766](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3766))
```
Comment (joshuarrrr) [inline CHANGELOG.md:168]:
```suggestion
- [Functional Tests] Remove unused listingTable methods ([#3766](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3766))

```

File: CHANGELOG.md [hunk 2]
```diff
@@ -233,6 +233,6 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
 - Update caniuse to fix failed integration tests ([#2322](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2322))
 - Update caniuse to 1.0.30001460 to fix failed integration tests ([#3538](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3538))
-
+- Removed functions which are no longer in use([#3766](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3766))
```
Comment (joshuarrrr) [inline CHANGELOG.md:236]: Our changelog process is confusing at the moment - but no need for a separate entry here.

```suggestion

```

File: src/setup_node_env/root/index.js [hunk 3]
```diff
@@ -29,10 +29,8 @@
  */
 
 var force = require('./force')(process.argv);
-
 var uid = process.getuid && process.getuid();
 var isRoot = require('./is_root')(uid);
-
```
Comment (joshuarrrr) [inline src/setup_node_env/root/index.js:35]: This isn't a problem, but it's an unrelated change - let's remove it from this PR.

File: test/functional/services/listing_table.ts [hunk 4]
```diff
@@ -65,34 +65,6 @@ export function ListingTableProvider({ getService, getPageObjects }: FtrProvider
       await searchFilter.click();
     }
 
-    private async getAllItemsNamesOnCurrentPage(): Promise<string[]> {
```
Comment (joshuarrrr) [inline test/functional/services/listing_table.ts:68]: Agreed that both of these methods are unused and can be deleted. I also verified that all the _other_ ListingTableProvider methods are still used.

**Review by joshuarrrr — COMMENTED** [review]
@Aigerim-ai The listing_table changes look great. Just a couple minor requested changes to the changelog and other stray file.

File: src/setup_node_env/root/index.js [hunk 5]
```diff
@@ -29,10 +29,8 @@
  */
 
 var force = require('./force')(process.argv);
```
Comment (joshuarrrr) [inline src/setup_node_env/root/index.js:31]:
```suggestion
var force = require('./force')(process.argv);

```

File: src/setup_node_env/root/index.js [hunk 6]
```diff
@@ -29,10 +29,8 @@
...
-
 var uid = process.getuid && process.getuid();
 var isRoot = require('./is_root')(uid);
```
Comment (joshuarrrr) [inline src/setup_node_env/root/index.js:33]:
```suggestion
var isRoot = require('./is_root')(uid);

```

**Comment by joshuarrrr** [comment]
@Aigerim-ai The DCO check is failing because some of your commits are not signed. Please ammend the commits with signatures or squash.

File: CHANGELOG.md [hunk 7]
```diff
@@ -233,6 +234,5 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
 - Update caniuse to fix failed integration tests ([#2322](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2322))
 - Update caniuse to 1.0.30001460 to fix failed integration tests ([#3538](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3538))
```
Comment (joshuarrrr) [inline CHANGELOG.md:236]:
```suggestion
- Update caniuse to 1.0.30001460 to fix failed integration tests ([#3538](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3538))

```
